libmscore/note.cpp

```diff
@@ -2898,7 +2898,7 @@ Element* Note::nextElement()
                   Element* next = nextInEl(e); // return next element in _el
                   if (next)
                         return next;
-                  else if (_tieFor)
+                  else if (_tieFor && !_tieFor->segmentsEmpty())
```

Comment:
Isn't it better to create standalone method with a clear name which incapsulates that check?

Reply:
You mean, to check for both the existance of the tieFor and the validity of it in one call?  I can do that, will add another commit.